src/flamegraph/mod.rs

```diff
@@ -345,21 +345,20 @@ where
     W: Write,
 {
     let mut reversed = StrStack::new();
-    let (mut frames, time, ignored, delta_max) = if !opt.no_sort && !opt.reverse_stack_order {
-        // Sort lines by default.
-        let mut lines: Vec<&str> = lines.into_iter().collect();
-        lines.sort_unstable();
-        merge::frames(lines)?
-    } else if opt.reverse_stack_order {
+    let (mut frames, time, ignored, delta_max) = if opt.reverse_stack_order {
         if opt.no_sort {
-            warn!("Input lines are always sorted when using --reverse. The --no-sort flag is being ignored.");
+            warn!("Input lines are always sorted when `reverse_stack_order` is `true`. The `no_sort` option is being ignored.");
```

Comment:
Just FYI, you can break strings using `\` at the end of the line :)

Reply:
I knew you could, I just wasn't thinking about it because I had line wrapping on.